great_expectations/checkpoint/checkpoint.py

```diff
@@ -407,8 +408,23 @@ def _run_validation(
                 f"Exception occurred while running validation[{idx}] of Checkpoint '{self.name}': {e.message}."
             ) from e
 
-    def self_check(self, pretty_print=True) -> dict:
-        # Provide visibility into parameters that Checkpoint was instantiated with.
+    @public_api
+    def self_check(self, pretty_print: bool = True) -> dict:
+        """
+        Method that is intended to provide visibility into parameters that Checkpoint was instantiated with.
```

Comment:
The first line needs to be on the line with the `"""` (sorry the linter will complain if not)

Reply:
https://dev.azure.com/great-expectations/great_expectations/_build/results?buildId=168479&view=logs&j=8f30d328-d63b-553b-4d9b-973ca5932bee&t=606bd092-5be4-5814-8ce5-2bd61e340414&l=467